src/explore-education-statistics-admin/src/pages/publication/PublicationMethodologiesPage.tsx

```diff
@@ -0,0 +1,500 @@
+import Link from '@admin/components/Link';
+import {
+  MethodologyStatusGuidanceModal,
+  MethodologyTypeGuidanceModal,
+} from '@admin/pages/publication/components/PublicationMethodologyGuidance';
+import usePublicationContext from '@admin/pages/publication/contexts/PublicationContext';
+import {
+  MethodologyRouteParams,
+  methodologySummaryRoute,
+} from '@admin/routes/methodologyRoutes';
+import {
+  PublicationRouteParams,
+  publicationAdoptMethodologyRoute,
+  publicationExternalMethodologyRoute,
+} from '@admin/routes/publicationRoutes';
+import methodologyService from '@admin/services/methodologyService';
+import publicationService, {
+  UpdatePublicationRequest,
+} from '@admin/services/publicationService';
+import Button from '@common/components/Button';
+import ButtonText from '@common/components/ButtonText';
+import FormattedDate from '@common/components/FormattedDate';
+import InfoIcon from '@common/components/InfoIcon';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import ModalConfirm from '@common/components/ModalConfirm';
+import Tag from '@common/components/Tag';
+import VisuallyHidden from '@common/components/VisuallyHidden';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import useToggle from '@common/hooks/useToggle';
+import React, { useState } from 'react';
+import { generatePath, useHistory } from 'react-router';
+
+const PublicationMethodologiesPage = () => {
+  const history = useHistory();
+  const { publicationId, onReload } = usePublicationContext();
+
+  // To ensure the methodologies are up to date when you switch to this tab
+  // we're re-fetching the publication here instead of using it from the context.
+  // This is not ideal and will be replaced by a call to just get the publication
+  // methodologies when an endpoint for this is ready - EES-3574 .
+  const { value: publication } = useAsyncHandledRetry(() =>
+    publicationService.getMyPublication(publicationId),
+  );
+
+  const [amendMethodologyId, setAmendMethodologyId] = useState<string>();
+  const [deleteMethodologyDetails, setDeleteMethodologyDetails] = useState<{
+    methodologyId: string;
+    amendment: boolean;
+  }>();
+  const [dropMethodologyId, setDropMethodologyId] = useState<string>();
+  const [
+    removingExternalMethodology,
+    toggleRemovingExternalMethodology,
+  ] = useToggle(false);
+  const [
+    showMethodologyTypeGuidance,
+    toggleMethodologyTypeGuidance,
+  ] = useToggle(false);
+  const [
+    showMethodologyStatusGuidance,
+    toggleMethodologyStatusGuidance,
+  ] = useToggle(false);
+
+  const handleRemoveExternalMethodology = async () => {
+    const updatedPublication: UpdatePublicationRequest = {
+      title,
+      contact: {
+        contactName: contact?.contactName ?? '',
+        contactTelNo: contact?.contactTelNo ?? '',
+        teamEmail: contact?.teamEmail ?? '',
+        teamName: contact?.teamName ?? '',
+      },
+      topicId,
+    };
```

Comment:
Similar to earlier, it's not a good idea to assign properties individually as we may incorrectly set things to null as part of the update.

Would suggest doing something like this instead:
```ts
const updatedPublication: UpdatePublicationRequest = {
  ... publication,
  externalMethodology: undefined
};
```